images/dvcr/werf.inc.yaml

```diff
@@ -1,15 +1,39 @@
 ---
-image: {{ $.ImageName }}-original-registry2
-final: false
-from: {{ .Images.BASE_CONTAINER_REGISTRY }}
----
 image: {{ $.ImageName }}
 from: {{ .Images.BASE_SCRATCH }}
 import:
-- image: {{ $.ImageName }}-original-registry2
-  add: /bin
+- image: {{ $.ImageName }}-builder
+  add: /container-registry-binary
   to: /bin
   after: install
-  includePaths:
-  - registry
-# Registry configuration is stored in configmap: templates/dvcr/configmap.yaml 
\ No newline at end of file
+# Registry configuration is stored in configmap: templates/dvcr/configmap.yaml 
+---
+{{- $version := "2.8.3" }}
+
+image: {{ $.ImageName }}-builder
+final: false
+fromImage: base-alt-p10
+shell:
+  install:
+    - |
+      apt-get update && apt-get install -y \
+      git openssh golang
+
+      export GOPATH=$(go env GOPATH)
+      export GOROOT=$(go env GOROOT)
+      export PATH=$PATH:$GOPATH/bin:$GOROOT/bin
```

Comment:
```suggestion
```

PATH update is needed after installing Go with `go install + go1.nn.nn download`. golang package installs go binary to /usr/bin.

Reply:
Fixed